src/status_im/ui/screens/profile/qr_code/views.cljs

```diff
@@ -35,15 +35,18 @@
                                                                             :height (.-height layout)}]))}
       (when (:width dimensions)
         [react/view {:style (styles/qr-code-container dimensions)}
-         [qr-code {:value (eip67/generate-uri (get contact qr-source) (when amount? {:value  amount}))
-                   :size  (- (min (:width dimensions)
-                                  (:height dimensions))
-                             80)}]])]
+         (when-let [value (eip67/generate-uri qr-value (when amount? {:value  amount}))]
```

Comment:
why does this happen in the view ?

Reply:
What? `nil` value? It should not but if it does app crash so this is defensive.

Reply:
No the generate-uri call, these things should be done in a subscription, views shouldn't contain any kind of computations

Reply:
That's not the scope of this PR so I would suggest to leave it out for now.